### PR TITLE
11037 inbox port

### PR DESCRIPTION
**Pull request by shanethmoore**
Merge of inbox API from 4.0.\* to 4.2

@Sergio-Mira Can you have a look at this?

### CODE REVIEW

**Comment by shanethmoore** [comment]
Original PR here for reference https://github.com/Swrve/swrve-android-sdk/pull/72

**Comment by Sergio-Mira** [comment]
:rocket:

File: SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java [hunk 1]
```diff
@@ -110,4 +113,12 @@
     Context getContext();
 
     C getConfig();
+
+    List<SwrveBaseCampaign> getCampaigns();
+
+    List<SwrveBaseCampaign> getCampaigns(SwrveOrientation orientation);
```
Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java:119]: Do these methods only get inbox campaigns?

Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java:119]: Yes, only inbox

Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java:119]: Should we rename it to getInboxCampaigns?

Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java:119]: I was instructed to not use the word Inbox as this feature is not really inbox, no UI yet :S

Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java:119]: getCampaigns is the wrong name for this api, so I don't think we should use that.

**Pull request merged**